src/server.js

```diff
@@ -51,6 +51,7 @@ function makeApp() {
 
 const app = makeApp();
 app.use(bodyParser.json());
+app.get(`${config.apiRoot}/applications/:id`, api.getApplicationStatus);
```

Comment:
Sounds good. I've left both in `api.js` since they're both attached to `/application`.